worker/earn.js

```diff
@@ -160,6 +162,10 @@ export function earn ({ models }) {
         await serialize(models,
           models.$executeRaw`SELECT earn(${earner.userId}::INTEGER, ${earnings},
           ${now}::timestamp without time zone, ${earner.type}::"EarnType", ${earner.id}::INTEGER, ${earner.rank}::INTEGER)`)
+        sendUserNotification(earner.userId, {
+          title: `you stacked ${numWithUnits(msatsToSats(earnings), { abbreviate: false })} in rewards`,
+          tag: 'EARN'
+        }).catch(console.error)
```

Comment:
Oh, there is one notification per user and type and not one notification per user only. So only the last notification shows up since the others before get replaced.
I have to sum the rewards up in the serviceworker (or sum them up here and only send one push notification) and I can include the rewards per type in the body then.